src/bindings/js/node/src/addon.cpp

```diff
@@ -41,6 +68,8 @@ Napi::Object init_module(Napi::Env env, Napi::Object exports) {
     init_class(env, exports, "ConstOutput", &Output<const ov::Node>::get_class, addon_data->const_output);
     init_class(env, exports, "PartialShape", &PartialShapeWrap::get_class, addon_data->partial_shape);
 
+    init_function(env, exports, "saveModel", save_model, addon_data->save_model);
```

Comment:
Function that blocks event loop must have postfix `Sync`:
```suggestion
    init_function(env, exports, "saveModelSync", save_model, addon_data->save_model);
```

Reply:
Should I also make `async` version?

Reply:
It is enough to have sync version as the solution of this issue